ui/app/templates/variables/variable/index.hbs

```diff
@@ -66,6 +66,11 @@
     {{/if}}
   </div>
 </h1>
+
+{{#if (or this.model.pathLinkedEntities.job this.model.pathLinkedEntities.group this.model.pathLinkedEntities.task)}}
+  <SecureVariableForm::RelatedEntities @job={{this.model.pathLinkedEntities.job}} @group={{this.model.pathLinkedEntities.group}} @task={{this.model.pathLinkedEntities.task}} />
+{{/if}}
```

Comment:
`blocking`: Can this get an acceptance? I'm pretty sure I'll break this in the future.